### PR TITLE
B 21169 int pr3

### PR DESCRIPTION
**Pull request by r-mettler**
## Summary

This PR moves the checking of the Alaska and Hawaii feature flags to the Handler level instead of the Service level. The service level seems to have issues with getting the correct status of the Feature Flag. I also temporarily added some logging to check to make sure the Boat and Mobile Home Feature Flags are being read properly since they are also checked at the Service level. 

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

For this test we need to make sure the FF is being read properly for AK and HI. 

1. Make sure the AK FF is on 
2. Login as an Office user or Customer and do an address search for an AK city or zip and make sure you see the results
3. Turn the AK FF off and verify you don't see any AK results
4. Repeat the above with the HI FF

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

**Review by antgmann — APPROVED** [review]
Bonne chance (but check gen code)

File: pkg/services/address/address_lookup_test.go [hunk 1]
```diff
@@ -10,11 +10,12 @@ func (suite *AddressSuite) TestAddressLookup() {
 	state := "NH"
 	postalCode := "03037"
 	county := "ROCKINGHAM"
+	excludedStates := [...]string{"AK", "HI"}
```
Comment (MInthavongsay) [inline pkg/services/address/address_lookup_test.go:13]: You may want to add tests to verify it's actually excluding AK and HI. I don't see any tests that does that.

Comment (r-mettler) [reply pkg/services/address/address_lookup_test.go:13]: added

**Review by MInthavongsay — APPROVED** [review]
Looks good.

File: pkg/handlers/ghcapi/addresses.go [hunk 2]
```diff
@@ -26,7 +28,36 @@ func (h GetLocationByZipCityStateHandler) Handle(params addressop.GetLocationByZ
 				return addressop.NewGetLocationByZipCityStateForbidden(), noOfficeUserIDErr
 			}
 
-			locationList, err := h.GetLocationsByZipCityState(appCtx, params.Search)
+			/** Feature Flag - Alaska - Determines if AK can be included/excluded **/
```
Comment (brianmanley-caci) [inline pkg/handlers/ghcapi/addresses.go:31]: If you wanted to add test for these ff scenarios you could. Just a thought.

File: pkg/handlers/internalapi/addresses.go [hunk 3]
```diff
@@ -87,7 +89,36 @@ func (h GetLocationByZipCityStateHandler) Handle(params addressop.GetLocationByZ
 				return addressop.NewGetLocationByZipCityStateForbidden(), noServiceMemberIDErr
 			}
 
-			locationList, err := h.GetLocationsByZipCityState(appCtx, params.Search)
+			/** Feature Flag - Alaska - Determines if AK can be included/excluded **/
+			isAlaskaEnabled := false
+			akFeatureFlagName := "enable_alaska"
```
Comment (brianmanley-caci) [inline pkg/handlers/internalapi/addresses.go:94]: I guess we don't have any kind of constant file for feature flags :(

**Review by brianmanley-caci — APPROVED** [review]
I added just a couple thoughts, but nothing to hold this up. Good job.

**Pull request merged**